File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/kotlin/F09.P2.kt
fun probe(): String {
    val xs = mutableListOf(2, 3, 1)
    val a = 4
    val b = 9
    // BEGIN PROBE F09.P2
    xs.sortDescending()
    val lt = a < b
    // END PROBE F09.P2
    return "$xs $lt"
}

fun main() {
    println(probe())
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/_superseded_run1/probes/python/F09.P2.py
def main() -> str:
    xs = [3, 1, 2]
    a = 4
    b = 9
# BEGIN PROBE F09.P2
    xs.sort()
    lt = a < b
# END PROBE F09.P2
    return f"{xs} {lt}"


print(main())
